projects/components/src/filter-bar/filter/builder/string-filter-builder.ts

```diff
@@ -2,22 +2,42 @@ import { FilterType } from '../../filter-type';
 import { UserFilterOperator } from '../filter-api';
 import { AbstractFilterBuilder } from './abstract-filter-builder';
 
-export class StringFilterBuilder extends AbstractFilterBuilder<string | undefined> {
-  public convertValue(value: unknown): string | undefined {
-    return value === undefined ? undefined : String(value);
+export class StringFilterBuilder extends AbstractFilterBuilder<string | string[] | undefined> {
```

Comment:
Should we be adding support to numeric filter too? number attributes should support in right?

Reply:
IN could make sense for numbers, especially since things like status code are numbers. I will add them in a follow up, but wanted to unblock myself for column filtering first.